src/main/ui/ultra-button.gd

```diff
@@ -8,7 +8,7 @@ Calculates the label text for displaying a player's fastest time for ultra mode.
 """
 func get_best_text(scenario_name: String) -> String:
 	var text := ""
-	var best_result := ScenarioHistory.get_best_result(scenario_name, "seconds")
+	var best_result := PlayerData.get_best_result(scenario_name, "seconds")
```

Comment:
In the absence of ScenarioHistory, this method should probably be called `get_best_scenario_result`

Reply:
Renamed; also renamed `get_best_scenario_rank` and other similar methods